memgpt/orm/tool.py

```diff
@@ -46,16 +46,16 @@ class Tool(SqlalchemyBase, OrganizationMixin):
     organization: Mapped["Organization"] = relationship("Organization", back_populates="tools", lazy="selectin")
 
     @classmethod
-    def read(cls, db_session: "Session", name: str) -> "Tool":
+    def read_by_name(cls, db_session: "Session", name: str) -> "Tool":
```

Comment:
@norton120 do we want to just do something like this, but also provide the actor? 
```
def read_by_name(cls, db_session: "Session", name: str, actor_id: str) -> "Tool": 
...
```